nuget.config

```diff
@@ -9,4 +9,4 @@
   <disabledPackageSources>
     <clear />
   </disabledPackageSources>
-</configuration>
+</configuration>
```

Comment:
nit: I have been typically adding newlines to each file I touch, and I think either VS or VS Code is also of the same persuasion. I think this is an old POSIX behavior which didn't process text that didn't end with a newline lol